### PR TITLE
Feature/video frame interpolation

### PR DESCRIPTION
**Pull request by zhangwlgq**
add rife-video-frame-interpolation and model

### CODE REVIEW

File: tests/pipelines/test_rife_video_frame_interpolation.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class RIFEVideoFrameInterpolationTest(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.task = Tasks.video_frame_interpolation
+        self.model_id = 'Damo_XR_Lab/cv_rife_video-frame-interpolation'
+        self.test_video = 'data/test/videos/video_frame_interpolation_test.mp4'
+
+    @unittest.skipUnless(test_level() >= 2, 'skip test in current test level')
```
Comment (wenmengzhou) [inline tests/pipelines/test_rife_video_frame_interpolation.py:21]: test_level() >= 0   do not skip this unittest

File: modelscope/pipelines/cv/rife_video_frame_interpolation_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+from modelscope.utils.constant import ModelFile, Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.video_frame_interpolation,
+    module_name=Pipelines.rife_video_frame_interpolation)
+class RIFEVideoFrameInterpolationPipeline(Pipeline):
```
Comment (wenmengzhou) [inline modelscope/pipelines/cv/rife_video_frame_interpolation_pipeline.py:33]: add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

File: tests/pipelines/test_rife_video_frame_interpolation.py (hunk 1 above)
Comment (zhangwlgq) [reply tests/pipelines/test_rife_video_frame_interpolation.py]: changed to test_level() >= 0

File: modelscope/pipelines/cv/rife_video_frame_interpolation_pipeline.py (hunk 2 above)
Comment (zhangwlgq) [reply modelscope/pipelines/cv/rife_video_frame_interpolation_pipeline.py:33]: added

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**